docs/src/tutorial/embedding.rst

```diff
@@ -69,19 +68,61 @@ The C ``main()`` function of your program could look like this:
 <https://docs.python.org/3/extending/extending.html#the-module-s-method-table-and-initialization-function>`_.)
 
 Instead of writing such a ``main()`` function yourself, you can also let
-Cython generate one into your module's C file with the ``cython --embed``
-option.  Or use the
-`cython_freeze <https://github.com/cython/cython/blob/master/bin/cython_freeze>`_
-script to embed multiple modules.  See the
-`embedding demo program <https://github.com/cython/cython/tree/master/Demos/embed>`_
-for a complete example setup.
-
-Be aware that your application will not contain any external dependencies that
-you use (including Python standard library modules) and so may not be truly portable.
-If you want to generate a portable application we recommend using a specialized
-tool (e.g. `PyInstaller <https://pyinstaller.org/en/stable/>`_
-or `cx_freeze <https://cx-freeze.readthedocs.io/en/latest/index.html>`_) to find and
-bundle these dependencies.
+Cython generate one into your module's C file with the ``cython --embed`` option.
```

Comment:
Here, I would take an opportunity and mention following points points:
1. `embedded.c` and `embedded.h` are generated by cython.
2. `embedded.c` file contains `__Pyx_main()` function which can be adjustd to call  `say_hello_from_python()` - here is snippet of code where added:
    ```c
    gcc -c embedded.c $(python3-config --cflags) -o embedded.o
    gcc embedded.o $(python3-config --ldflags --embed)  -o embedded
    ```
 3. after compilation `embedded` executable is created:
     ```
     $ ./embedded
     Hello from Python!
     ```